src/hotspot/share/oops/stackChunkOop.inline.hpp

```diff
@@ -135,8 +135,8 @@ inline bool stackChunkOopDesc::is_in_chunk(void* p) const {
 }
 
 bool stackChunkOopDesc::is_usable_in_chunk(void* p) const {
-#if (defined(X86) || defined(AARCH64)) && !defined(ZERO)
-  HeapWord* start = (HeapWord*)start_address() + sp() - frame::sender_sp_offset;
+#if (defined(X86) || defined(AARCH64) || defined(RISCV64)) && !defined(ZERO)
+  HeapWord* start = (HeapWord*)start_address() + sp() - frame::metadata_words;
```

Comment:
This looks platform independent now, doesn't it? I think the cpp conditional can be removed. You can leave it also to the PPC64 port as I'll touch that line again. As you like.

Reply:
I think I will leave it to the PR for PPC64. Thanks.